cmd/telegraf/telegraf_windows.go

```diff
@@ -96,10 +99,7 @@ func runAsWindowsService(inputFilters, outputFilters, aggregatorFilters, process
 		}
 		os.Exit(0)
 	} else {
-		winlogger, err := s.Logger(nil)
 		if err == nil {
```

Comment:
looks like you left the error handling behind, but there's no call.

Reply:
updated.